graphql_compiler/scripts/generate_test_sql/utils.py

```diff
@@ -0,0 +1,71 @@
+import random
+
+
+random.seed(0)
+
+CREATE_VERTEX = 'create vertex '
+CREATE_EDGE = 'create edge '
+SEPARATOR = '__'
+
+
+def get_uuid():
+    """Return a pseudorandom uuid."""
+    random_digits = ''.join([str(random.randint(0, 9)) for _ in range(32)])  # nosec
```

Comment:
`get_uuid()` is simply `str(UUID(int=random.randint(0, 2**128 - 1)))`, no need to format the uuid manually